### ISSUE TITLE
False Positive on org.apache.logging.log4j:log4j-api-2.14.1.jar

### ISSUE THREAD

**Issue by bleloch** [dependency-check/DependencyCheck#3875]
`log4j-api-2.14.1.jar` has been flagged as susceptible to the recent Log4j vulnerability, but based on sources [here](https://www.randori.com/blog/cve-2021-44228/) and [here](https://github.com/mubix/CVE-2021-44228-Log4Shell-Hashes), only `log4j-core*.jar` from versions 2.0 to 2.14.1 is affected, not the API jar.

```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:6.2.2:aggregate (default) on project parent: 
[ERROR] 
[ERROR] One or more dependencies were identified with vulnerabilities: 
[ERROR] 
[ERROR] log4j-api-2.14.1.jar: CVE-2021-44228
```

No CPE has been raised yet for Log4j 2.14, only up to 2.13 according to [this page](https://nvd.nist.gov/vuln/detail/CVE-2021-44228/cpes?expandCpeRanges=true). I am not sure why this is the case as it's widely reported that 2.14 is also impacted.

False positive on library log4j-api-2.14.1.jar 
```xml
<dependency>
    <groupId>org.apache.logging.log4j</groupId>
    <artifactId>log4j-api</artifactId>
    <version>2.14.1</version>
</dependency>
```

**Comment by mprins** [comment]
Please check existing open en closed issues before adding, duplicates #3868

**Comment by bleloch** [comment]
Apologies, closed.
